public/json/KeychronK6F1HelperShortcuts.json

```
@@ -69,6 +69,26 @@
           ]
         }
       ]
+    },
+    {
+      "description": "Home to F11",
+      "manipulators": [
+        {
+          "type": "basic",
+          "from": {
+            "key_code": "home",
+            "modifiers": {
+              "mandatory": []
+            }
```

Remove empty `modifiers`?

```suggestion
            "key_code": "home"
```